packages/esm-patient-medications-app/src/order-basket/order-basket-search/order-basket-search-results.component.tsx

```diff
@@ -13,16 +13,29 @@ export interface OrderBasketSearchResultsProps {
   searchTerm: string;
   setSearchTerm: (value: string) => void;
   onSearchResultClicked: (searchResult: OrderBasketItem, directlyAddToBasket: boolean) => void;
+  searchInputRef: RefObject<HTMLInputElement>;
 }
 
 export default function OrderBasketSearchResults({
```

Comment:
Don't pass the ref inside another component if the ref is not required.

Instead create a function
```
function focusOnSearchInput() {
  ref.current.focus()
}
```
and then pass this function in as a prop, which can then be called in the handleClick.

Reply:
For passing refs, please refer: https://react.dev/reference/react/forwardRef